src/main/java/com/smartystreets/api/us_reverse_geo/Result.java

```diff
@@ -0,0 +1,59 @@
+package com.smartystreets.api.us_reverse_geo;
+
+import com.google.api.client.util.Key;
+
+public class Result {
+    //region [ Fields ]
+
+    @Key("address")
+    private Address address;
+
+    @Key("coordinate")
+    private Coordinate coordinate;
+
+    @Key("distance")
+    private double distance;
+
+    @Key("latitude")
+    private double latitude;
+
+    @Key("longitude")
+    private double longitude;
```

Comment:
Remove this field from the Result class. It should only be provided on the Coordinate class